components/dialog/stories/dialog.stories.js

```diff
@@ -46,39 +115,183 @@ export default {
 			},
 			control: "boolean",
 		},
+		heroImageUrl: {
+			name: "Hero Image",
+			type: { name: "string" },
+			description: "Adds a cover image to the header of a dialog.",
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "file", accept: ".svg,.png,.jpg,.jpeg,.webc" },
+			if: { arg: "layout", eq: "default" },
+		},
 	},
 	args: {
 		rootClass: "spectrum-Dialog",
-		isDismissable: true,
-		showModal: false,
+		hasFooter: true,
+		footer: "Do not show this message again.",
+		isDismissible: false,
 		isOpen: true,
+		showModal: false,
+		size: "m",
+		layout: "default",
+		hasCheckbox: true,
 	},
 	parameters: {
 		actions: {
-			handles: ["click .spectrum-Dialog button"],
+			handles: [],
 		},
 		docs: {
 			story: {
+				// inline: false,
```

Comment:
I can add the `todo` comment from main to make it a little clearer why this is comments out.